docs/src/test/java/docs/http/javadsl/Http2Test.java

```diff
@@ -0,0 +1,32 @@
+package docs.http.javadsl;
+
+import java.util.concurrent.CompletionStage;
+import akka.japi.Function;
+import akka.actor.ExtendedActorSystem;
+import akka.http.javadsl.HttpsConnectionContext;
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.model.HttpResponse;
+import akka.stream.Materializer;
+
+//#bindAndHandleAsync
+import akka.http.javadsl.Http;
+import static akka.http.javadsl.ConnectHttp.toHostHttps;
+
+//#bindAndHandleAsync
+
+class Http2Test {
+  void testBindAndHandleAsync() {
+    Function<HttpRequest, CompletionStage<HttpResponse>> asyncHandler = null;
+    Materializer materializer = null;
+    ExtendedActorSystem system = null;
+    HttpsConnectionContext httpsConnectionContext = null;
+
+    //#bindAndHandleAsync
+    Http http = new Http(system);
+    http.setDefaultServerHttpContext(httpsConnectionContext);
```

Comment:
Remove these two lines. I think it should only be something like this (could be split up into multiple lines):

```scala
Http.get(system).bindAndHandleAsync(asyncHandler, ConnectHttp.toHostHttps("127.0.0.1", 8443).withCustomHttpsContext(httpsConnectionContext), materializer);
```

The `ConnectHttp` API is a bit weird for the server side.